modules/pixfutureBidAdapter.md

```diff
@@ -0,0 +1,29 @@
+# Overview
+
+```
+Module Name: PixFuture Bid Adapter
+Module Type: Bidder Adapter
+Maintainer: admin@pixfuture.net
+```
+# Description
+
+Module that connects to PixFuture demand sources
+
+# Test Parameters
+```
+var adUnits = [{
+   {
+  "pageUrl": "some referer URL",
+  "bidId": "608601dbbacc5",
+  "auctionId": "4842f47b-a12b-4eed-8eac-fc71d37a5e30",
+  "transactionId": "b4004949-b099-48eb-ba04-902f4fffcaca",
+  "adUnitCode": "26335x300x250x14x_ADSLOT88",
+  "sizes": [
+    [300, 250],
+    [300, 600]
+  ],
+  {
+  "params": {
+    "pix_id": "777"
+  }];
```

Comment:
This config example doesn't work, please correct it. You can easily verify your config example in `integrationExamples` (you could use `/gpt/hello_world.html`).

Reply:
The config example has been fixed